pkg/apply/event/event.go

```diff
@@ -1,3 +1,6 @@
+// Copyright 2020 The Kubernetes Authors.
```

Comment:
It think the missing license headers are my fault. But can we fix them in a separate PR?